tests/test_aggregators.py

```diff
@@ -1,9 +1,43 @@
 import pytest
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from steering_vectors.aggregators import (
+    linear_aggregator,
+    logistic_aggregator,
+    mean_aggregator,
+    pca_aggregator,
+)
 import torch
-from steering_vectors.aggregators import mean_aggregator, pca_aggregator
 from torch.nn.functional import cosine_similarity
 
 
+def test_linear_aggregator() -> None:
+    pos = torch.randn(64, 100)
+    neg = -1 * torch.randn(64, 100)
```

Comment:
This test could be made more robust by generating pos and neg similar to how is done in the PCA test, where we have a known delta we're trying to recover with some noise, and verifying that this linear regression is able to recover it. Something like:

```python
    delta = torch.randn(100)
    pos = torch.randn(64, 100)
    noise = torch.randn(64, 100) * 0.2
    neg = pos - delta.unsqueeze(0) + noise
```

Ideally, this aggregator should get close to estimating the `delta`, similar to the PCA test if it's working correctly.

Reply:
Done with [7ba358f](https://github.com/steering-vectors/steering-vectors/pull/22/commits/7ba358f8229cbe662932a7d35cffe92cefbf6e28) (the linear regression aggregator wasn't able to handle the noise well though, so I removed it for that one)